core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1163,7 +1163,10 @@ oppia.directive('schemaBasedEditor', [function() {
       onInputFocus: '='
     },
     templateUrl: 'schemaBasedEditor/master',
-    restrict: 'E'
+    restrict: 'E',
+    controller: function() {
+      document.execCommand('enableObjectResizing', false, false);
```

Comment:
(textAngular is the open source library which we use for the RTE, so, since this is an RTE thing, it seems sensible to keep it there.)